backstory/src/main/java/com/google/sps/vision/VisionManager.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.vision;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Interface for the Image Analytics Manager.
+ *
+ * VisionManager Manages the gathering and packaging of Vision API image analytics.
+ * A VisionManager object represents an image annotated with analytics from Vision API.
+ *
+ * The image is represented as the byte data in the rawImageData byte array, and the annotated
+ * analytics are represented as labelAnnotations, a list of Vision EntityAnnotation objects, which
+ * each represent individual labels.
+ */
+
+public interface VisionManager {
```

Comment:
Nit: Interfaces are by default public so it isn't necessary to tag it so.

Reply:
I talked to AnneMarie about this too, we are going to include the public tag in the interface to make the code easier to understand. In case somebody does not understand that the default for an interface method is that they are public. We could omit it though, if you think there is a good reason for it.

Reply:
Oh I revised mine to not include public after Joe commented so and that's pushed to the code base at the moment, and AnneMarie's current PR's omit the public so the change may be the move here.